rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -535,3 +556,53 @@ SettingsDialog::SettingsDialog(wxWindow* parent)
 		config.write(out.c_str(), out.size());
 	}
 }
+
+void SettingsDialog::OnModuleListItemToggled(wxCommandEvent &event)
+{
+	lle_module_list[fmt::ToUTF8(event.GetString())] = chbox_list_core_lle->IsChecked(event.GetSelection());
+}
+
+void SettingsDialog::OnSearchBoxTextChanged(wxCommandEvent &event)
+{
+	// helper to preserve alphabetically order while inserting items
+	int item_index = 0;
+
+	if (event.GetString().IsEmpty())
+	{
+		for (auto& i : lle_module_list)
+		{
+			if (i.second)
+			{
+				chbox_list_core_lle->Check(chbox_list_core_lle->Insert(i.first, item_index));
+				item_index++;
+			}
+
+			if (!i.second)
+			{
+				chbox_list_core_lle->Check(chbox_list_core_lle->Insert(i.first, chbox_list_core_lle->GetCount()), false);
+			}
+		}
+	}
+
+	chbox_list_core_lle->Clear();
+
+	wxString search_term = event.GetString().Lower();
+	item_index = 0;
+
+	for (auto& i : lle_module_list)
+	{
+		if (fmt::FromUTF8(i.first).Find(search_term) != wxString::npos)
+		{
+			if (i.second)
+			{
+				chbox_list_core_lle->Check(chbox_list_core_lle->Insert(i.first, item_index));
+				item_index++;
+			}
+
+			if (!i.second)
```

Comment:
I'm so dumb right now, hadn't slept for like 25 hours, let me fix this real quick.